chrome_manifest_v3/class_app.js

```
@@ -0,0 +1,848 @@
+
+    // If the string is partial, then it was the right-side key:
+    if (retn.slice(-1) === '-') {
+      retn += 'right';
+    }
+
+    return retn;
+  }
+
+  url_add_var(url_in = '', var_in = '') {
+    let add = '';
+    if (url_in && url_in.length && var_in && var_in.length) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)